### PR TITLE
UI Development

### PR DESCRIPTION
**Pull request by shashjar**
Makes all of adjustments from last year's scouting app to this year's. UI has been changed to include all metrics currently identified for each of Match Metadata, Autonomous (Sandstorm Period), TeleOp Period, and Endgame. UI has been viewed and tested, all is functional. Additional changes yet to be made include using jquery to separate the different sections of the scouting app into tabs so that the scout can switch between them at any point in the match, as well as adding the metric of which side a team scores on during autonomous.

### CODE REVIEW

**Review by roshanr10 — CHANGES_REQUESTED** [review]
Separate jQuery Tabs Please

**Comment by shashjar** [comment]
jQuery tabs will be added, merge for now?

**Comment by roshanr10** [comment]
UI goes hand in hand w/ the tabs, see my comments in issue #2... should be easy enough but I'd like to see how it looks when simulated for the screen resolution before we finalize this

**Comment by shashjar** [comment]
@roshanr10  Please review the UI, let me know with any suggestions for changes. Tabs are finalized, overflow has been set to scroll, tabs have not yet been colored/styled. If no issues, I'm good with merging now.

**Comment by roshanr10** [comment]
I'd like to wrap up the tab styling today before we merge into master.

File: app/www/js/main.js [hunk 1]
```diff
@@ -65,14 +95,61 @@ $(applicationCache)
...
+        "Reckless Driving" : $("#reckless-driving").is(":checked") ? 1 : 0,
+        "Not Present" : $("#not-present").is(":checked") ? 1 : 0,
+        "Disabled" : $("#disabled").is(":checked") ? 1 : 0,
+        "Robot Failure" : $("#robot-failure").is(":checked") ? 1 : 0,
+        "Top Heavy" : $("#top-heavy").is(":checked") ? 1 : 0,
+        "Foul" : $("#foul").is(":checked") ? 1 : 0,
+        "Card" : $("#card").is(":checked") ? 1 : 0
+    };
+    
+    // TODO: Clear Fields
```
Comment (roshanr10) [inline app/www/js/main.js:166]: @ShashJar can you implement this? should be fairly straightforward, numeric fields default to 0, checkboxes to false, and text fields to ""

**Comment by roshanr10** [comment]
Pending for Basic Web Usage:
- [ ] Clear on Submit @ShashJar 
- [ ] basic output view (match and average) @roshanr10 
- [x] off-line data @roshanr10 
- [x] disable arrows on number input (https://www.thatstevensguy.com/programming/disable-arrows-on-number-inputs/) @roshanr10 

Links for self:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
https://datatables.net/reference/option/data

**Comment by roshanr10** [comment]
@shashjar sorry buddy those changes won’t work, did you test..?

See http://api.jquery.com/val/#val2 for more help on setting textbox values in Jquery and https://stackoverflow.com/a/17420580 for the checkbox equivalent.

**Review by roshanr10 — APPROVED** [review]

**Pull request merged**